### PR TITLE
Updated enet server test.cpp

### PR DESCRIPTION
**Pull request by wadawdaw**
Okey so i added FIXED Skin color ande /gem so u can get gems cuz many developers wanted that or kids k
i too fixed the grass in the world i hop this is usefull btw 
Go sub my channel NabzGT if u wanna get some codes soon!

### CODE REVIEW

**Comment by Symzad** [comment]
Add /gem to /help

File: enet server test/enet server test.cpp [hunk 1]
```diff
@@ -2508,6 +2508,19 @@ int _tmain(int argc, _TCHAR* argv[])
...
+						{
+							GamePacket p = packetEnd(appendInt(appendString(createPacket(), "OnSetBux"), atoi(str.substr(5).c_str())));
+							ENetPacket * packet = enet_packet_create(p.data,
+								p.len,
+								ENET_PACKET_FLAG_RELIABLE);
+
+							enet_peer_send(peer, 0, packet);
+							delete p.data;
+							continue;
+
```
Comment (Mempler) [inline enet server test/enet server test.cpp]: Here are alot of whitespaces

File: enet server test/enet server test.cpp [hunk 2]
```diff
@@ -2508,6 +2508,19 @@ int _tmain(int argc, _TCHAR* argv[])
 								}
 							}
 						}
+						else if (str.substr(0, 5) == "/gem ") //gem if u want flex with ur gems!
```
Comment (Mempler) [inline enet server test/enet server test.cpp]: Great idea!

**Comment by wadawdaw** [comment]
SO I CHANGED /GEM to bottom of /help if u wanted that im done of it

**Comment by Mempler** [comment]
No no no, we mean add it to the help command.

Like all other commands.

just add /gem

**Pull request merged**